docs/README.md

```diff
@@ -300,6 +300,8 @@ The "Show More" button that is used to load the results of the next search page.
 The "Show Previous" button that is used to load the results of the previous search page. This block is not rendered if there is no previous page. Does not need any prop.
 - **`search-products-count-per-page` block**
 Shows the product count per search page. Does not need any prop.
+- **`search-products-progress-bar` block**
```

Comment:
There is something wrong with the markdown formating. It might be missing `}` in lines 101 and 121.

Reply:
Fixed! Thanks!